src/local/butler/weights.py

```diff
@@ -316,6 +316,45 @@ def _aggregate_fuzzer_jobs(
   _print_stats(others, total_weight)
 
 
+def _set_fuzzer_job_weight(
+    fuzzer: str,
+    job: str,
+    weight: float,
+) -> None:
+  """Sets the matching FuzzerJob's weight to the given value."""
+  fuzzer_jobs = list(
+      data_types.FuzzerJob.query(data_types.FuzzerJob.fuzzer == fuzzer,
+                                 data_types.FuzzerJob.job == job))
+
+  if not fuzzer_jobs:
+    print('No matching FuzzerJob entries found for ' +
+          f'fuzzer {fuzzer} and job {job}')
+    return
+
+  if len(fuzzer_jobs) > 1:
+    print('Bailing out! Multiple FuzzerJob entries found for ' +
+          f'fuzzer {fuzzer} and job {job}: {fuzzer_jobs}')
+    return
+
+  fj = fuzzer_jobs[0]
```

Comment:
Nit: I'm not against obvious abbreviations like this, but it's never done in CF ;-)
Can you call it fuzzer_job please?

Reply:
Done! Also did it for 2 other places in the file.